doc/user/content/sql/create-sink/kafka.md

```diff
@@ -405,7 +405,7 @@ running `CREATE SINK`, observe the following guidance:
 | Progress topic | Replication factor  | Your choice, based on your durability requirements.
 | Progress topic | Compaction          | We recommend enabling compaction to avoid accumulating unbounded state. Disabling compaction may cause performance issues, but will not cause correctness issues.
 | Progress topic | Retention           | **Must be disabled.** Enabling retention can cause Materialize to violate its [exactly-once guarantees](#exactly-once-processing).
-
+| Progress topic | Tiered Storage      | We recommend disabling tiered storage to enable more aggressive data compaction. Fully compacted data requires minimal storage, typically only tens of bytes per sink, making it cost-effective to maintain directly on local disk.
```

Comment:
Trivial nit (feel free to ignore): changing "enable" to "allow for" -- just to avoid cognitive dissonance of disabling to enable

```suggestion
| Progress topic | Tiered Storage      | We recommend disabling tiered storage to allow for more aggressive data compaction. Fully compacted data requires minimal storage, typically only tens of bytes per sink, making it cost-effective to maintain directly on local disk.
```